app/assets/stylesheets/components/_theme.css.scss

```diff
@@ -86,7 +86,8 @@ $theme-font-family: "brandon-grotesque";
 
 /** Spacing **/
 $default-spacing: 1rem;
-
+$default-padding: 1rem;
```

Comment:
there is no such things as a `$default-padding` and a `$default-card-padding`, you don't need these